test/built-ins/Proxy/deleteProperty/targetdesc-is-configurable-target-is-not-extensible.js

```diff
@@ -0,0 +1,40 @@
+// Copyright (C) 2019 Aleksey Shvayka. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+esid: sec-proxy-object-internal-methods-and-internal-slots-delete-p
+description: >
+    Throw a TypeError exception if trap result is true, targetDesc is configurable,
+    and target is not extensible.
+info: |
+    [[Delete]] (P)
+
+    ...
+    13. Let extensibleTarget be ? IsExtensible(target).
+    14. If extensibleTarget is false, throw a TypeError exception.
+    ...
+features: [Proxy, Reflect]
+---*/
+
+var trapCalls = 0;
+var target = {
+  prop: 1,
+};
+
+var p = new Proxy(target, {
+  deleteProperty: function(t, prop) {
+    trapCalls++;
+    return true;
+  },
+});
+
+Object.preventExtensions(target);
+
+assert.throws(TypeError, function() {
+  Reflect.deleteProperty(p, "prop");
+});
+assert.sameValue(trapCalls, 1);
+
+assert.throws(TypeError, function() {
+  Reflect.deleteProperty(p, "nonExistent");
```

Comment:
here is the trick:

```
11. If targetDesc is undefined, return true.
```

Before the isExtensible check. So this should not throw, but return `true`.

I thought I said that in the first review but I did not, my fault.

Reply:
ccc3072

It is totally my fault: copied & pasted assertion without any thinking.